tests/common.py

```diff
@@ -7,4 +7,9 @@
 sys.path.append(package_directory)
 
 # Module to be tested
-import romaine  # noqa
+import romaine
+
+
+# Utility for getting an initialised parser.
+def get_romaine_parser():
+    return romaine.Core().parser
```

Comment:
if we change the parser instantiation thing this could be `return romaine.Core.Parser()`